.github/workflows/release.yaml

```diff
@@ -1,32 +1,27 @@
 name: "Release"
 on:
-  pull_request:
+  push:
     branches:
-      - main
-    types:
-      - closed
+      - 'release/**'
+      - 'main'
```

Comment:
```suggestion
```

afaics we don't need the release workflow for pushes to main, but only for the `release` branches?

Reply:
Is this for listen when someone push to main right? this happens when the release PR is merged.

Reply:
Ah, yes, you're right. So we should remove the `release/**` then?
The release workflow should run only after a push to `main`, if the `CHANGELOG.md` got modified

Reply:
Make sense? :D

Reply:
Yes, I think we don't needed.

Reply:
Done!